src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpImportServiceDAO.scala

```diff
@@ -39,6 +38,22 @@ class HttpImportServiceDAO(implicit val system: ActorSystem, implicit val materi
     }
   }
 
+  override def getJob(workspaceNamespace: String, workspaceName: String, jobId: String)(implicit userInfo: UserInfo): Future[ImportServiceListResponse] = {
+    // get single job from import service
+    val importServiceUrl = FireCloudDirectiveUtils
+      .encodeUri(s"${FireCloudConfig.ImportService.server}/$workspaceNamespace/$workspaceName/imports/$jobId")
+
+    userAuthedRequest(Get(importServiceUrl))(userInfo) flatMap { importServiceResponse =>
+      importServiceResponse.status match {
+        case StatusCodes.OK => Unmarshal(importServiceResponse).to[ImportServiceListResponse]
+        case _ =>
+          extractResponseBody(importServiceResponse.entity) flatMap { respBody =>
+            Future.failed(new FireCloudExceptionWithErrorReport(ErrorReport(importServiceResponse.status, respBody)))
+          }
+      }
+    }
+  }
```

Comment:
the `case _` is necessary here because the call to Import Service via `userAuthedRequest` is successful even when Import Service returns a 404 or 500 or whatever … yes, we successfully got a response from Import Service; that response happens to be an error.